sensors/imu/hmc5883l.py

```diff
@@ -1,7 +1,8 @@
-from sensors.imu.imu_sensor import IMUSensor
```

Comment:
the name hmc58831.py, could it be renamed for a better meaning

Reply:
Good point, I'll rename all of them